package.json

```diff
@@ -47,7 +54,7 @@
   },
   "homepage": "https://github.com/libp2p/js-interfaces#readme",
   "dependencies": {
-    "@types/bl": "4.1.0",
+    "@types/bl": "^4.1.0",
```

Comment:
probably should be "^5.0.0" now, Alex updated it and it was released yesterday, bl@5 has been out for a week or so too thanks to Alex as well

Reply:
bl seems like a weird thing to use at the interface level, perhaps it's something we can pull out in future?

It's only used [in one place](https://github.com/libp2p/js-libp2p-interfaces/blob/master/src/pubsub/index.js#L348) and from the type def it looks like it's only necessary because the exported BufferList can't be treated as a Uint8Array based on the type heirachy.